packages/network/src/NetworkCanvas.js

```diff
@@ -60,6 +61,7 @@ const NetworkCanvas = props => {
 
     const theme = useTheme()
     const getNodeColor = useNodeColor(nodeColor)
+    const getNodeComponent = useNodeComponent(nodeComponent)
```

Comment:
This isn't used currently, you can limit this to the SVG implementation, or add support for canvas, but you'll have to adjust the TS definitions and prop-types, also a custom node component for canvas would be a function receiving a 2D canvas context as the first argument and the _props_ as the second one. The swarmplot package use this approach if you're curious, and given you're willing to implement it of course :)

Reply:
I have updated this PR with this suggestion. Let me know if there is any improvements for the the Canvas implementation 🙏